kolibri/utils/cli.py

```diff
@@ -144,7 +144,7 @@ def get_cext_path(dist_path):
 # Add path for c extensions to sys.path
 get_cext_path(sys.path[-1])
 try:
-    import cryptography # noqa
+    import cryptography  # noqa
```

Comment:
You can also use this pattern to avoid the extra commit:

```
# edit ./file
git add file
git commit - "Adding something"
git push origin branch-name
# edit ./file
git add file
git commit --amend  # Add to previous commit
git push origin branch-name -f  # Force-push to your own feature branch, that's okay
```

Reply:
Thank you! That really helps